acrobat/blocks/dc-converter-widget/dc-converter-widget.js

```diff
@@ -208,18 +222,29 @@ export default async function init(element) {
 
   const isReturningUser = window.localStorage.getItem('pdfnow.auth');
   const isRedirection = /redirect_(?:conversion|files)=true/.test(window.location.search);
-  const preRenderDropZone = !isReturningUser && !isRedirection;
+  let isLimitExhausted = false;
+  const isExportVerb = VERB.includes('pdf-to');
+  const isCreateVerb = VERB.includes('to-pdf');
+  if (exhLimitCookieMap[VERB]) {
+    isLimitExhausted = cookies.includes(exhLimitCookieMap[VERB]);
+  } else if (isExportVerb) {
+    isLimitExhausted = cookies.includes(exhLimitCookieMap['pdf-to']);
+  } else if (isCreateVerb) {
+    isLimitExhausted = cookies.includes(exhLimitCookieMap['to-pdf']);
+  }
 
   const verbIncludeList = ['compress-pdf', 'fillsign', 'sendforsignature', 'add-comment',
     'delete-pages', 'reorder-pages', 'split-pdf', 'insert-pdf', 'extract-pages', 'crop-pages', 'number-pages'];
 
+  const preRenderDropZone = verbIncludeList.includes(VERB) || (!isReturningUser && !isRedirection);
```

Comment:
we can get rid of verbIncludeList and the check here. We were only doing that to avoid processing for returning users to L2 verbs (aside from compress)

```suggestion
  const preRenderDropZone = !isLimitExhausted && !isRedirection;
```

Reply:
Updated